searx/shared/redisdb.py

```diff
@@ -26,26 +26,31 @@
 from searx import get_setting
 
 
-logger = logging.getLogger('searx.shared.redis')
-_client = None
+OLD_REDIS_URL_DEFAULT_URL = 'unix:///usr/local/searxng-redis/run/redis.sock?db=0'
+"""This was the default Redis URL in settings.yml."""
 
+_CLIENT = None
+logger = logging.getLogger('searx.shared.redisdb')
 
-def client():
-    global _client  # pylint: disable=global-statement
-    if _client is None:
-        # not thread safe: in the worst case scenario, two or more clients are
-        # initialized only one is kept, the others are garbage collected.
-        _client = redis.Redis.from_url(get_setting('redis.url'))
-    return _client
 
+def client() -> redis.Redis:
+    return _CLIENT
 
-def init():
+
+def initialize():
```

Comment:
Why is this function renamed? / at least the doc-string in line 11 needs to be modified

https://github.com/searxng/searxng/blob/ee4475ff4b7f1d7d687606664c3b1dd4cafe5a99/searx/shared/redisdb.py#L10-L13